docs/getting-started/products/ylockers/yprisma/yprisma-faq.md

```diff
@@ -0,0 +1,83 @@
+  * Vault depositors are "all in" on yPRISMA.
+* **Gas Fees:**
+  * Direct stakers must manually claim their mkUSD stablecoin rewards - which requires gas and can be expensive.
+  * Vault depositors have their mkUSD stablecoin rewards claimed, swapped and reinvested into the yv-yPRISMA vault each week. And Yearn covers the gas.
+* **Composability:**
+  * Direct stakers do not receive a receipt token in their wallet.
+  * Vault depositors receive back a yv-yPRISMA token into their wallet which has the potential to be accepted as collateral across DeFi.
+
+## Can yPRISMA users vote with their yPRISMA?
+
+No. User voting is not part of the yLockers system, instead...
+
+* Gauge voting is automated to optimize for the maximum weekly bribe yield.
+* Governance voting is handled by a team of core contributors on behalf of veYFI holders.
+
+## Why is the yPRISMA peg currently below 1:1?
+
+It is normal for a yToken such as yPRISMA (or any liquid locker token) to trade at a discount to the governance token it is built upon. The strength of the peg is dependent on the supply and demand of each yToken.
+
+Each yLocker has been designed to create an attractive yToken that offers value to users and provides incentives to attract liquidity. However, we cannot control the market price of our yTokens.
+
+## What is yvmkUSD-A?
+
+yvmkUSD-A is a Yearn V3 vault that allows yield to auto-compound. Currently, there is no active strategy for compounding, but a strategy is nearly completed and will undergo reviews soon.
+
+## What are the fees associated with yPRISMA?
+
+When you stake yPRISMA, a 10% performance fee is applied. Yield accumulates in the receiver contract throughout the week. The yield is distributed once per week, at which point 10% is sent to the Yearn treasury.
+
+## Where does the yield come from and how does it flow?
+
+The yield for yPRISMA stakers comes from two main sources:
+
+* Protocol Fees: Fees generated from the Prisma protocol
+* Bribes: Incentives provided for voting power
+
+All these yields are claimed to the yPRISMA fee receiver at 0x76DF88Aa8711822472Cb40Ed8c972A461A20ecdc
+
+At the beginning of each week, the collected yield is converted to mkUSD (Prisma Stablecoin), deposited into the yvmkUSD-A vault, and then distributed to stakers. This process occurs once per week, and the yield is streamed evenly over the entire week.
+
+## Why would I claim my emissions as yPRISMA?
+
+There are several key reasons to claim your emissions as yPRISMA instead of locking yourself.
+
+1. When minting yPRISMA, you get to utilize Yearn's boost, which is among the highest of total Prisma accounts.
+1. yPRISMA is minted 1:1 against your max locked claim amount.
+1. yPRISMA is fully transferrable and can be swaped on DEXes like Curve, or staked for benefits in the ecosystem.
+
+## How can I claim my emissions as yPRISMA?
```

Comment:
Similar point here about reenforcing that this pertains only to users manually farming prisma's receivers through their website

Reply:
see comment above.